### PR TITLE
Update change log - release v2.3.0

### PR DESCRIPTION
**Pull request by w-vi**

### CODE REVIEW

**Comment by pksunkara** [comment]
Looks like we need to do this whenever we are releasing.

**Comment by w-vi** [comment]
Yes

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 ## Master
 
+
+## 2.3.0
+
 ### Bug Fixes
 
 * Parameters in Refract output will no longer contain a `content` value
-  when there is no sample value.  
+  when there is no sample value.
```
Comment (kylef) [inline CHANGELOG.md]: These double spaces we're done on purpose to trigger a new line in markdown.

screen shot 2016-04-15 at 16 46 14

vs

screen shot 2016-04-15 at 16 46 22

**Comment by w-vi** [comment]
Sorry, I didn't realize it ... significant trailing whitespace sucks. I use extra line for this purpose, not used to this.

**Comment by kylef** [comment]
@w-vi Extra white line would result in breaking out of the list item wouldn't it?

**Comment by w-vi** [comment]
@kylef You are right it adds extra line in between the items. Now it is clear that I don't use markdown much :)

**Pull request merged**